app/models/account.rb

```diff
@@ -185,8 +185,12 @@ def subscribed?
     !subscription_expires_at.blank?
   end
 
-  def followers_domains
-    followers.reorder(nil).pluck('distinct accounts.domain')
+  # Domains potentially used for PuSH callback URLs of followers' instances
+  def followers_subscription_domains
+    domains = followers.reorder(nil).pluck('distinct accounts.domain')
+    remote_urls = followers.reorder(nil).pluck('distinct accounts.remote_url')
```

Comment:
I'll push a slightly optimized version of this, which should probably be ok for about a thousand followers across a few hundreds remote domains, but not much more.